packages/slice-machine/src/modules/modal/types.ts

```diff
@@ -1,5 +1,6 @@
 export enum ModalKeysEnum {
   LOGIN = "LOGIN",
+  NEW_VERSION = "NEW_VERSION",
```

Comment:
I think this key name should be coherent with the Modal name to make it more explicit, so I will change the modalName from "updateModal" => "newVersionModal"